BCH-Wunder-Project-team-4/Wunder.io
# Footer with form and links

## Link to ticket:

[[Link to the ticket]](https://trello.com/c/daaE0gxa)

Huge thanks to [Jesse](https://github.com/jekku123) for guiding  me

## Link to feature environment:

[Add link to feature environment]

## Changes proposed in this PR:

Added footer with subscribe to newsletter feature and links to other resources in footer

## How to test:

1. Try to subscribe (do not check anything just press subscribe)
2. enter e-mail and try omt
3. check terms and try omt (you should see a warning toast)
4.  check something (you should see success toast )

## Best practices:

<details>
<summary><h3>Accessibility:</h3></summary>
<p>
This project must support WCAG accessibility level AA <em>(edit this according to the requirements of your project)</em>. To ensure this standard is met, remember to:

- Perform automated checks using a tool such as Wave or SiteImprove.
- Test keyboard navigation: are all parts of the UI navigable using only the keyboard? Is the tab order logical? Can popups, menus etc be dismissed with the escape key?
- Test responsiveness, scaling and text reflow.
- Make sure no accessibility issues exist on either desktop or mobile views.
- If you have time, test with a screen reader such as VoiceOver (macOS), NVDA (Windows), or Orca (Linux).

Use the [Accessibility Testing Cheat Sheet](https://intra.wunder.io/info/accessibility-group/accessibility-testing-cheat-sheet) for information on how to run these tests.
</p>
</details>
